### ISSUE TITLE
Session1 q3 topics to cover

### ISSUE THREAD

**Issue by natbutter** [Sydney-Informatics-Hub/geopython#5]
Q3
--
Are there   any particular topics/code/libraries/datasets you want to cover, or are   looking forward to, or that we covered and want us to elaborate on? Feel free   to provide links or go into detail if appropriate.      We will see if we can incorporate some of your suggestions in the later   sessions (or schedule some one-on-one time for a crash course).
{"ImportId":"QID3_TEXT"}
you guys rock
My main interest is geological mapping
Working with large datasets
mainly machine learning.
Statistical analysis of geospatial data   would be most useful for me
Plotting data (maps, histograms,   stereonets)
 
Nothing specific yet
 
I'm excited to see LAS and SEGY in the   'coming soon....'
I'm curious about adding interactivity to   plots and maps and so on with (something like) Bokeh... and I'm excited to   learn about TensorFlow and the like.
Obviously external libraries such as   "numpy" and the plotting library are key resources. Are you able to   spend some time describing the strengths and weaknesses of key libraries and   how we find out what is in each
I have a general interest in all things   Python, but keen to understand spatial graphics and data analysis (data   cleanup, looking for correlations, relationships etc prior to machine   learning, for example
'-	Are   there libraries designed to work on bathymetry (xyz / grid files)     -	Are there libraries to automatically analyse raster images (e.g.,   photographs of cores)     -	With deep learning: is it possible to create training data from scratch?   How to classify and label training data?     -	Clustering: is it possible to do clustering on multi-dimensional data to   identify relevant parameters to create groups of geological objects? (for   example, if we have measured geological objects â€“ width, height, length,   asymmetryâ€¦ - but cannot group them based on only 2 parameters)     Thanks again for the great course!
looking forward to lasio and dealing with   shape files :)
Looking forward to lasio.
Happy to go with course content.
 
 
 
not at this stage, will email you   whenever this comes up during the next sessions. thanks.

**Comment by natbutter** [comment]
Stale issue, hopefully fixed at some point as the notes have changed since now.
